great_expectations/checkpoint/actions.py

```diff
@@ -1118,7 +1122,7 @@ class CloudNotificationAction(ValidationAction):
 
     def __init__(
         self,
-        data_context: "DataContext",
+        data_context: DataContext,
```

Comment:
This should be `AbstractDataContext` but this module isn't being type-checked yet.

I'm anticipating we may be renaming `AbstractDataContext` -> `DataContext` so I've left it as `DataContext` to prevent extra churn in this module.

If we decide against this, this will need to change.